src/v2/Apps/Algolia/algoliaRoutes.tsx

```diff
@@ -0,0 +1,60 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "v2/System/Router/Route"
+import { graphql } from "react-relay"
+
+const AlgoliaApp = loadable(
```

Comment:
I imagine that after this is promoted out of a demo state we'll rename these components (and the route) to just search, yah?

Reply:
Yes, that's right